Apps/System/systemapp.c

```diff
@@ -140,6 +154,76 @@ static void systemapp_window_unload(Window *window)
     menu_destroy(s_menu);
 }
 
+static void window_exit_handler(ClickRecognizerRef recognizer, void *context)
+{
+    window_stack_pop(true);  
+}
+
+static void about_window_load(Window *window)
+{
+    Layer *window_layer = window_get_root_layer(s_about_window);
+    GRect bounds = layer_get_bounds(window_layer);
+
+    status_bar = status_bar_layer_create();
+    status_bar_layer_set_separator_mode(status_bar, StatusBarLayerSeparatorModeDotted);
+
+    status_bar_layer_set_colors(status_bar, PBL_IF_COLOR_ELSE(GColorRed,GColorBlack), GColorWhite);
+
+    status_bar_layer_set_text(status_bar, "About RebbleOS");
+
+    s_about_scroll = scroll_layer_create(bounds);
+    scroll_layer_set_click_config_onto_window(s_about_scroll, window);
+
+	s_aboutCanvas_layer = layer_create(bounds);
```

Comment:
caution mixed indentation (we aren't great about this internally righ tnow so I won't block merge on this)